CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Fixed
+- sensuctl now supports the http_proxy, https_proxy, and no_proxy environment
```

Comment:
`https_proxy` is fine; is it `https_proxy` or `HTTPS_PROXY` or both? Same question for `no_proxy`. Is `all_proxy` available? Most programs follow curl/libcurl's convention; see the "ENVIRONMENT" section of `curl(1)` for details.

Reply:
Answered my own question; resty uses [`http.ProxyFromEnvironment`](https://golang.org/pkg/net/http/#ProxyFromEnvironment).

Reply:
The Go documentation mentions the capitalized versions of these environment variables, but I tested it with the lower-cased version, since that's what curl uses.

The Go documentation does not mention `all_proxy`, so I assume it's not supported.

Reply:
I believe resty simply passes through this configuration to the underlying http client.